astropy/table/table.py

```diff
@@ -1841,6 +1841,13 @@ def show_in_notebook(
         html += jsv.ipynb(tableid, css=css, sort_columns=sortable_columns)
         return HTML(html)
 
+    @deprecated(
+        "6.1",
+        pending=True,
```

Comment:
Why is this not a proper deprecation?

Reply:
I think the idea was to take this one a bit slower to gather user inputs.

Reply:
I don't think most user would see `pending=True` warning.  Could you please open a follow-up issue on when you think we can remove `pending=True` here? Thanks!

Reply:
Might also want to post to astropy-dev once you have this follow-up issue. Any further discussions on a merged PR would be easily overlooked.